app/views/trial_feedbacks/show.html.slim

```diff
@@ -0,0 +1,39 @@
+h1= t_title(:show)
+
+.table-responsive.m-t-30.m-b-20
+  table.table.table-no-border-top
+    tbody
+      - if policy(TrialFeedback).superadmin_priviledges?
+        tr
+          td= t_model(Volunteer)
+          td= link_to @trial_feedback.volunteer.contact.full_name, volunteer_path(@trial_feedback.volunteer)
+      - if @trial_feedback.assignment?
+        tr
+          td= t_model(Assignment)
+          td= link_to @trial_feedback.feedbackable.to_label, assignment_path(@trial_feedback.feedbackable)
+        tr
+          td= t_model(Client)
+          td= link_to @trial_feedback.feedbackable.client.contact.full_name, client_path(@trial_feedback.feedbackable.client)
+      - else
+        tr
+          td= t_model(GroupOffer)
+          td= link_to @trial_feedback.feedbackable.to_label, group_offer_path(@trial_feedback.feedbackable)
+      tr
+        td Text
+        td= word_wrap(@trial_feedback.body) if @trial_feedback.body
+      - if policy(TrialFeedback).superadmin_priviledges?
+        tr
+          td Erstellt von
+          td= link_to @trial_feedback.author.full_name, @trial_feedback.author
+        tr
+          td Eingesehen von
+          td
+            - if @trial_feedback.reviewer
+              = link_to @trial_feedback.reviewer.full_name, @trial_feedback.reviewer if @trial_feedback.reviewer
+            - else
+              = button_to 'Angeschaut', polymorphic_path([@volunteer, @trial_feedback.feedbackable, @trial_feedback],
+                { trial_feedback: { volunteer_id: @trial_feedback.volunteer.id, feedbackable_id: @trial_feedback.feedbackable.id, id: @trial_feedback.id } }),
```

Comment:
could we put this maybe in a helper method? 3 lines for a button action makes it hard to read